deployments/fargate/README.md

```diff
@@ -0,0 +1,143 @@
+- Set environment variable `SPLUNK_CONFIG` to the default configuration file path
+  (`/etc/otel/collector/fargate_config.yaml`).
+- Optional:
+  - Assign a list of metrics you want excluded to environment variable `METRICS_TO_EXCLUDE`.
+  - Assign a list of images whose metrics you want excluded to environment variable
+    `IMAGES_TO_EXCLUDE`.
+
+## Custom Configuration
+Include the following in your container definition for the Collector in Fargate:
+- Specify the Collector image.
+- Set environment variable `SPLUNK_CONFIG` to the custom configuration file path.
+- Optional:
+  - Use environment variable `SPLUNK_CONFIG_YAML` instead of `SPLUNK_CONFIG` to specify
+    the configuration YAML directly (No file needed).
+  - Use extension `ecs_observer` to discover targets.
+  - Enable ECS read-only permissions in task role for `ecs_observer`.
+  - Filter `ecs_observer` targets to be within task.
+
+Note that your custom configuration file needs to be in a volume attached to the task.
+Also, `ecs_observer` is currently limited to Prometheus targets.
+
+### ecs_observer
+Extension
+[Amazon Elastic Container Service Observer](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/ecsobserver#amazon-elastic-container-service-observer)
+(`ecsobserver` or `ecs_observer`) can discover metrics targets in running tasks, filtered
+by service names, task definitions and container labels. `ecs_observer` requires the 
+read-only permissions below. You can add the permissions to the task role by adding them
+to a customer-managed policy that is attached to the task role.
+```text
+ecs:List*
+ecs:Describe*
+```
+
+Below is a custom configuration example in which the `ecs_observer` is configured to find
+Prometheus targets in cluster `lorem-ipsum-cluster`, region `us-west-2`, where the task arn
+pattern is `^arn:aws:ecs:us-west-2:906383545488:task-definition/lorem-ipsum-task:[0-9]+$`.
+The results are written to file `/etc/ecs_sd_targets.yaml`. The `prometheus` receiver is
+configured to read targets from the results file. Note that the values for `access_token`
+and `realm` are read from environment variables `SPLUNK_ACCESS_TOKEN` and `SPLUNK_REALM`
+respectively, which must be specified in your container definition.
+
+```yaml
+extensions:
+  ecs_observer:
+    refresh_interval: 10s
+    cluster_name: 'lorem-ipsum-cluster'
+    cluster_region: 'us-west-2'
+    result_file: '/etc/ecs_sd_targets.yaml'
+    task_definitions:
+      - arn_pattern: "^arn:aws:ecs:us-west-2:906383545488:task-definition/lorem-ipsum-task:[0-9]+$"
+        metrics_ports: [9113]
+        metrics_path: /metrics
+receivers:
+  signalfx:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'lorem-ipsum-nginx'
+          scrape_interval: 10s
+          file_sd_configs:
+            - files:
+                - '/etc/ecs_sd_targets.yaml'
+processors:
+  batch:
+  resourcedetection:
+    detectors: [ecs]
```

Comment:
this doesn't have `system` yet fargate config does -- why?

Reply:
I'll remove the system detector from the default config